### PR TITLE
Issue #6 & #7: better csv output

### PR DESCRIPTION
**Pull request by IvyaThalenmourn**
Added course id, course name, and assignment name to csv output.
Modified csv download name to include course name, assignment name, and the current date.

### CODE REVIEW

File: src/helpers.py [hunk 1]
```diff
@@ -1,103 +1,85 @@
...
+    out = [d for d in some_list if _matches_dict_key_val(d, match_key, match_val)][0]
+    return(out)
 
 
 def __get_assessment_criteria_scores(assessment_rating):
+    # Create a dictionary which includes assessment criteria and points assigned
     criteria = assessment_rating.get("criterion").get("description")
-
     ratings_dictionary = {criteria: assessment_rating.get("points")}
-
```
Comment (alisonmyers) [inline src/helpers.py:89]: Can you explain this change to me?

File: src/util.py [hunk 2]
```diff
@@ -1,41 +1,33 @@
...
+"""Local utility module that handles printing error and success messages.
 
-last edit:
-Monday, January 12, 2020
+This module exports the following functions:
+    print_error - prints and formats an error message.
+    print_success - prints and formats a success message.
 """
+# authors:
+# @markoprodanovic
```
Comment (alisonmyers) [inline src/util.py:8]: you can add your name here too :)

**Review by alisonmyers — CHANGES_REQUESTED** [review]
Please review comments and see feedback. 
Overall looks great and seems to work well, will approve request after comments addressed 👍

File: src/helpers.py (hunk 1 above)
Comment (IvyaThalenmourn) [reply src/helpers.py:89]: Most of the code in this function was commented out and unused, so I deleted the commented out part.

File: src/util.py (hunk 2 above)
Comment (IvyaThalenmourn) [reply src/util.py:8]: Name or github username? Also shouldn't your name be here too?

Comment (alisonmyers) [reply src/util.py:8]: up to you - I haven't added anything to the util file

File: src/helpers.py (hunk 1 above)
Comment (alisonmyers) [reply src/helpers.py:89]: didn't even notice the comments

**Comment by alisonmyers** [comment]
closes #6 #7 #8 #10 #11

**Pull request merged**